### PR TITLE
checking out what this diff is gonna look like

### PR DESCRIPTION
**Pull request by GabeLoins**
@vqtran hey so now we handle sessions. 

I added an input text box in the UI that you enter your user id, then we grab your user id from the session object on the backend.

the only tricky part is getting your user id initially (not sure if/where amazon publishes this). a more reliable method is just looking in the sessions table in the sessions db to see what user id has been auto inserted for you. not sure if the test terminal and the echo itself will have different ids for you (my guess is they will not). my id is the one beginning with AF7Q, so don't use that one!

as usual, the ui looks like shit

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,5 @@ airportdata/
 
 *.swp
 *.swo
+
+public/assets/bundle.js
```
Comment (GabeLoins) [inline .gitignore]: yeah I know we have to do other stuff to fix this but I didn't feel like it right now

File: src/main/java/echoquery/utils/VisualizationUtil.java [hunk 2]
```diff
@@ -38,4 +43,26 @@ public static String getDisplayText() {
...
+        Statement statement = conn.createStatement();
+        statement.executeUpdate("insert into sessions values ('" + 
+            cleanId(userId) + "', 'EchoQuery. What do you want?');");
+      } catch (SQLException e) {
+        // silently ignore - we already have this value in the db
+      }
+  }
+  
+  // removes the text at the beginning of the id
+  public static String cleanId(String userId) {
```
Comment (GabeLoins) [inline src/main/java/echoquery/utils/VisualizationUtil.java]: the ids are by default formatted like amazon.user.KJEFLKDSJBDFSK or something like that so I snip off the first two default words

File: .gitignore (hunk 1 above)
Comment (GabeLoins) [inline .gitignore]: ignore the above

**Comment by vqtran** [comment]
Just gave this a super quick pass but thought it would be about time it got merged, go for it!

**Pull request merged**